spec/features/user_signin_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+feature "Reset password" do
+  let(:user) { create :user }
+  scenario "User can reset password" do
+    visit new_user_session_path
+    click_link "Forgot your password?"
+    fill_in :user_email, with: user.email
+    click_button "Reset My Password"
+    expect(page).to have_content t('devise.passwords.send_instructions')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.